physionet-django/user/templates/user/credential_application.html

```diff
@@ -24,8 +24,8 @@ <h2>Recommendations for a timely review</h2>
   <hr>
   <p>The following steps will help to ensure a timely review of your application:</p>
   <ul>
-    <li>Listing a reference is always helpful. Remember to prompt your reference to expect an email about your application.</li>
-    <li>It is also helpful if your reference is a registered user and they have linked their ORCID iD to their profile. Please provide the email associated with their account.</li>
+    <li>All users should list a reference. Please provide their academic or institutional email. Remember to prompt your reference to expect an email about your application.</li>
```

Comment:
Please could you change this to: "You are required to provide a reference, who we may contact with regard to your application."

Reply:
Sorry Lama, should have thought more about this before making the suggestion. It should be "whom"!

![Screenshot 2023-01-23 at 5 19 54 PM](https://user-images.githubusercontent.com/822601/214163182-68febee1-0093-4b58-aeb6-222f142f19b3.png)

Reply:
no problem, will update it